usr/lib/greenboot/check/required.d/01_repository_dns_check.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Are these just bought straight over from the old version? If so it would be useful to document that in the commit log. If they're not changed from the old version and they're in the same location it might cause some merge issues but we'll want to make sure we preserve the git history.

Reply:
done!